openwisp_users/utils.py

```diff
@@ -1,10 +1,14 @@
+import logging
+
 from django.conf import settings
 
 if 'reversion' in settings.INSTALLED_APPS:  # pragma: no cover
     from reversion.admin import VersionAdmin as BaseModelAdmin
 else:
     from django.contrib.admin import ModelAdmin as BaseModelAdmin
 
+logger = logging.getLogger(__name__)
```

Comment:
why is this being added again?
I had to remove it, it's causing issues, see https://github.com/openwisp/openwisp-users/commit/2e98874636621aa1d5406e293d63112de87c8c20